README.md

```diff
@@ -112,6 +112,13 @@ To run tests, run the following command:
 - Twitter: [@Twitter](https://twitter.com/home)
 - LinkedIn: [@LinkedIn](https://www.linkedin.com/in/diana-beki/)
 
+👤 **Uwimbabazi Dorcas**
+
+- GitHub: [@Dorcas126](https://github.com/Dorcas126)
+- Twitter: [@Dorcasuw](https://twitter.com/Dorcasuwi)
+- LinkedIn: [Uwimbabazi Dorcas](https://www.linkedin.com/in/uwimbabazi-dorcas-956a5a226/)
+
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work so far but it would be nice if you have a way for the user to create a recipe I am not able to create a recipe in your app :fearful: Kindly make a way so a user can create a recipe in your app :+1: because you would deploy this app and provide a live link and users will use the live link so they won't be able to create a recipe 😨